dipy/reconst/shore.py

```diff
@@ -679,7 +679,7 @@ def l_shore(radial_order):
 
 
 def n_shore(radial_order):
-    "Returns the angular regularisation matrix for SHORE basis"
+    """Returns the angular regularisation matrix for SHORE basis"""
```

Comment:
BTW, the `l_shore`, and `n_shore` methods are exactly the same. I have not found the equations in the paper.

Reply:
Maybe @maurozucchelli has an answer concerning l_shore and n_shore?